pkg/api/aim/dashboards.go

```diff
@@ -197,12 +230,18 @@ func DeleteDashboard(c *fiber.Ctx) error {
 	}
 	if err := database.DB.
 		Select("dashboards.id").
-		Joins("LEFT JOIN apps ON dashboards.app_id = apps.id").
+		InnerJoins(
+			"App",
+			database.DB.Where(
+				&database.App{
```

Comment:
@jgiannuzzi it is a bit strange I know but:
```
Base: database.Base{
 						IsArchived: false,
 					},
```
leads Python tests failing. Can't say right now why.

Reply:
can you please investigate what the Python tests do that our integration tests don't and add it to integration tests?

Reply:
@jgiannuzzi can we do that in scope of other task? Im not sure that it could be super fast, since I never touch the Python tests part?